src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,56 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Returns Starbucks Locations as an array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
```

Comment:
The class doesn't technically only return the Starbucks locations since it also does some file parsing. It would probably be more accurate to change this comment to something along the lines of this servelet handles parsing and serving Starbucks data and move the comment about returning starbucks data down to the doGet maybe.